packages/app-degree-pages/src/components/DetailPage/components/RequiredCourse/index.js

```diff
@@ -1,85 +1,119 @@
 /* eslint-disable react/prop-types */
 import PropTypes from "prop-types";
 import React from "react";
-import Styled from "styled-components";
-
-import { Button } from "../../../../../../components-core/src/components/Button";
 
 /** @typedef {import('../../../../core/types/detail-page-types').RequiredCoursesProps} RequiredCoursesProps */
 
-const ButtonGroup = Styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
-  gap: 1.5rem;
-  & a.btn {
-    margin: 0;
-  }
-`;
+const CHANGEMAJOR_URL = "https://changemajor.apps.asu.edu/";
+const ONLINE = "ONLINE";
 
 /**
  *
  * @param {RequiredCoursesProps} props
  * @returns
  */
 function RequiredCourse({
-  concurrentDegreeMajorMaps = "",
   onlineMajorMapURL = "",
-  majorMapOnCampusArchiveURL,
+  majorMapOnCampusArchiveURL = "",
```

Comment:
Love the improvements to readability, @maycomayco . I'd like to request one change - the `majorMapOnCampusArchiveURL` variable name should be `majorMapOnCampusURL` since it's not an archive link. At one point in the past that was a point of confusion, so let's remove it now.